posthog/version.py

```diff
@@ -1,4 +1,4 @@
-VERSION = "3.11.1"
+VERSION = "3.12.1"
```

Comment:
```suggestion
VERSION = "3.12.0"
```

the patch has to be always 0 when there's a new minor

Reply:
Oops, duh thank you